packages/orchestration/src/util/filtering.ts

```diff
@@ -43,7 +43,7 @@ export function buildAzureContentSafetyFilter(
  */
 export function buildLlamaGuardFilter(
   ...categories: [LlamaGuardCategory, ...LlamaGuardCategory[]]
-): LlamaGuard38BFilterConfig {
```

Comment:
[q] Since they named it very specific to the model (also with a `type`), if in the future they have another llama config (4 12B), then we might have a naming problem.

Reply:
Yes. I was initially thinking if we could make the function extensible/generic to allow users to construct all types of Llama filters. But after azure filter types, can't say with confidence if that'll be possible.